Frontend/moodipy/src/components/OnboardingPage/OnboardingPage.vue

```diff
@@ -9,14 +9,38 @@
       v-model="message"
       placeholder="Type your current mood"
     />
+    <div v-if="validEntry == false">
+      *Please enter a word or a group of word
+    </div>
     <br />&nbsp;<br />
-    <button id="getStarted">Get Started</button>
+    <button id="getStarted" @click="saveInputEmotion(message)">
+      Get Started
+    </button>
   </div>
 </template>
 
 <script>
 export default {
   name: "StartPage",
+  data() {
+    return {
+      emotion: "",
```

Comment:
rename to initialEmotion since we'll have 2 emotions  one for initial and one for final